lldb/test/API/lang/swift/nested_generic_class/TestSwiftNestedGenericClass.py

```diff
@@ -5,12 +5,26 @@
 import unittest2
 
 
-class SwiftNestedGenericClassTest(TestBase):
+class TestSwiftNestedGenericClassTest(TestBase):
```

Comment:
`TestSwiftNestedGenericClass`?

Reply:
it also doesn't need to match the file name, it can be `TestCase`.